content/kotlin/concepts/math-methods/terms/asin/asin.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'asin()'
+description: 'Returns the arcsine of a numeric argument.'
+subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+tags:
+  - 'Double'
+  - 'Float'
+  - 'Methods'
+  - 'Functions'
+catalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`asin()`** function returns the arcsine of a number.
+
+## Syntax
+
+```pseudo
+asin(x)
+```
+
+The function accepts one mandatory parameter `x`. It should be a numeric value in the range between `-1` and `1` (inclusive). The argument can be of `Double`, `Float`, or `Long` datatype.
```

Comment:
```suggestion
The function accepts one mandatory parameter `x`. It should be a numeric value in the range between `-1` and `1` (inclusive). The argument can be of `Double`, and `Float` datatype.
```
Long is a numeric data type, it doesn't make as much sense to use it as an argument for the asin() function.